config/manager/default-config/deployment.yaml

```diff
@@ -36,7 +36,8 @@ spec:
           env:
             - name: NPM_CONFIG_USERCONFIG
               value: /opt/app-root/src/.npmrc.dynamic-plugins
-          image: <RELATED_IMAGE_backstage> # will be replaced with the actual image quay.io/janus-idp/backstage-showcase:next
+          #image: <RELATED_IMAGE_backstage> # will be replaced with the actual image quay.io/janus-idp/backstage-showcase:next
+          image: quay.io/janus-idp/backstage-showcase:latest
```

Comment:
To be consistent, I'd suggest using the same default value for the `RELATED_IMAGE_backstage` env var set in the CSV. It uses a different tag: `quay.io/janus-idp/backstage-showcase:next`. See https://github.com/janus-idp/operator/blob/main/config/manager/manager.yaml#L77

Reply:
As I remember, some time ago we were suggested to not to use the "next" tag, otherwise I have not prefs.

Reply:
Agreed. So let's use `latest` in the CSV as well.

Reply:
by who? 

why would you want the `main` = 0.2/1.2 operator to install the previous stable `1.1.x` = 0.1/1.1 hub?